common/lib/dependabot/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Dependabot
-  VERSION = "0.127.0"
+  VERSION = "0.128.0"
```

Comment:
Did we not have an actual `0.127.1` bump?

Reply:
This was just a side effect of the way I branched off main to create this PR which creted a conflict. Should be fixed now 👍🏻